mne/decoding/transformer.py

```diff
@@ -248,6 +248,12 @@ def inverse_transform(self, X, y=None):
 class PSDEstimator(TransformerMixin):
     """Compute power spectrum density (PSD) using a multi-taper method
 
+    This structures data so that it can be easily incorporated into
+    scikit-learn pipelines. It relies heavily on the multitaper_psd
```

Comment:
Agreed - I wasn't sure if that's something people would want but can do that for sure